.changelog/11027.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:bug

```

Comment:
Should this be `release-note:breaking-change` instead for a removal of a deprecated parameter?

Reply:
Ya, that seems appropriate.